### PR TITLE
Fix some flake8 errors

### PR DESCRIPTION
**Pull request by pquentin**
Hello Dag Wieers,

You don't seem to be opposed to PEP 8 changes (did not see a single issue or pull request mentioning this), so I thought this pull request could be helpful. I believe that codebases that follow PEP 8 are easier to read simply because they are more consistent with other Python code.

This does not fix all issues, but fixes most of them. It currently ignores (see the tox.ini file):
  * E262 (inline comment should start with ‘#‘)
  * E265 (block comment should start with ‘#‘)
  * E501 (line too long (82 > 79 characters))
  * F821 (undefined name)

If you think the pull request is too big, I can split it in several parts. If there are some PEP 8 errors that you don't want to fix, I can remove the fixes too. To test the pull request, you can run "flake8 unoconv".

Thank you for unoconv.

### CODE REVIEW

**Comment by pquentin** [comment]
Yay, the tests pass! I did not break basic functionality.

@dagwieers, I understand if you don't want to merge this. There are lots of reasons not to : you don't want to break existing PRs, this is too much to review, you don't care about PEP 8, you don't have enough time...

But could you please tell me what to expect here? Thanks!

**Comment by pquentin** [comment]
@dagwieers gentle ping

**Comment by pquentin** [comment]
@dagwieers Anything I can do about this? Do you think it is a bad idea?

**Comment by pquentin** [comment]
@dagwieers If reviewing pull requests is too much work for you, why don't you recruit maintainers? Thanks for unoconv!

**Comment by dagwieers** [comment]
Sorry, I remember looking at this at least once.

I prefer not to merge this because it will break other pull requests. I do care about PEP8, but prefer fixes as part of real changes only. Also, I don't care about some of the PEP8 rules (e.g. 80 columns etc...).

That said, there are some real fixes in your pull-request that I do want to retain now.

File: unoconv [hunk 1]
```diff
@@ -956,16 +967,16 @@ class Convertor:
...
+                # the images in the file. In the resulting document, the rest of the images did not appear. Cycling
+                # through all the image references in the document seems to force OO to actually load them. Found
                 # some helpful guidance in this thread:
                 # https://forum.openoffice.org/en/forum/viewtopic.php?f=30&t=23909
                 # Ideally we would like to have the option to embed the images into the document, but I have not been
                 # able to figure out how to do this yet.
                 graphObjs = document.GraphicObjects
                 for i in range(0, graphObjs.getCount()):
-                    graphObj = graphObjs.getByIndex(i)
+                    graphObjs.getByIndex(i)
```
Comment (dagwieers) [inline unoconv]: Valid fix !

File: unoconv [hunk 2]
```diff
@@ -1144,7 +1156,7 @@ class Listener:
             resolver = self.svcmgr.createInstanceWithContext("com.sun.star.bridge.UnoUrlResolver", self.context)
             product = self.svcmgr.createInstance("com.sun.star.configuration.ConfigurationProvider").createInstanceWithArguments("com.sun.star.configuration.ConfigurationAccess", UnoProps(nodepath="/org.openoffice.Setup/Product"))
             try:
-                unocontext = resolver.resolve("uno:%s" % op.connection)
+                resolver.resolve("uno:%s" % op.connection)
```
Comment (dagwieers) [inline unoconv]: Seems like a valid fix too.

File: unoconv [hunk 3]
```diff
@@ -1213,7 +1228,7 @@ def die(ret, msg=None):
 
         ### LibreOffice processes may get stuck and we have to kill them
         ### Is it still running ?
-        if ooproc.poll() == None:
+        if ooproc.poll() is None:
```
Comment (dagwieers) [inline unoconv]: Valid fix.

**Comment by pquentin** [comment]
Thanks for the review! I'll consider submiting another PR with those changes. (By the way, even if you call those valid fixes, they don't change any functionality, just like the other changes.)

**Pull request closed without merging**